paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/LinkTest.kt

```diff
@@ -473,13 +466,6 @@ internal class LinkTest {
                 response.testBodyFromFile("consumer-session-lookup-success.json")
             }
 
-            networkRule.enqueue(
-                method("POST"),
-                path("/v1/consumers/sessions/lookup"),
-            ) { response ->
-                response.testBodyFromFile("consumer-session-lookup-success.json")
-            }
-
```

Comment:
These reductions in calls are expected. We were making two lookup calls when fetching the account status, once in `PaymentSheetLoader` and once in `LinkHandler`. We have merged these into one call in `PaymentSheetLoader`.